scrapers/shagmag.yml

```diff
@@ -0,0 +1,33 @@
+name: 'SHAGMAG'
+galleryByURL:
+  - action: scrapeXPath
+    url:
+      - shagmag.com/issues/
+    scraper: galleryScraper
+
+xPathScrapers:
+  galleryScraper:
+    gallery:
+      Title: //h1[@class="t-Page-Heading-1"]/text()
+      URL: //link[@rel="canonical"]/@href
+      Studio:
+        Name:
+          fixed: 'SHAGMAG'
+      Date:
+        selector: //h1[@class="t-Page-Heading-1"]/text()
+        postProcess:
+          - replace:
+            # There's one post per month so just assume the 1st so we can get a usable date
+            - regex: ^
+              with: '1 '
+          - parseDate: 2 January 2006
```

Comment:
Sorry I didn't notice this until it was merged,
but I found the official published date on the page source,
though I don't have an example where it's **not** the first day of the month.
```yml
      Date:
        selector: //script[@type="application/ld+json"]
        postProcess:
          - replace:
              - regex: ^.+[,{]"datePublished":"(.+?)T[\d:+-]+"[},].+$
                with: $1
          - parseDate: 2006-01-02
```

Reply:
Do you want to make a PR about it ? i kinda overlooked it too since its a magazine and they usually have fixed release dates anyway.

Reply:
Oh nice. I couldn't see any dates on the page and didn't think to check within the script elements.

Reply:
If it's mostly the first day of each month anyway it doesn't need the extra complexity.
So no PR from me.